controllers/grid/users/reviewer/form/ReviewerForm.inc.php

```diff
@@ -241,12 +242,16 @@ function fetch($request) {
 			'reviewDueDate' => __('reviewer.submission.reviewDueDate'),
 			'submissionReviewUrl' => __('common.url'),
 			'reviewerUserName' => __('user.username'),
+			'contextName' => $context->getLocalizedName(),
```

Comment:
Since some variables are going to be available to all email templates, I suspect it would make sense to put this into a superclass so that we don't have to re-code them for every specific case.